src/pages/menuUpload/supplierMaster/SupplierMasterPage.tsx

```diff
@@ -16,42 +18,92 @@ import AddItemButton from 'shared/buttons/AddItemButton';
 import Dialog from 'shared/dialog/Dialog';
 import {
   deleteSupplierMasterRequest,
-  getSupplierMasterRequest,
   postSupplierMasterRequest,
   putSupplierMasterRequest,
 } from 'stores/actions/supplierMaster/supplierMasterActions';
 import { RootState } from 'stores/reducers/rootReducer';
+import refreshAccessToken from 'stores/sagas/utils';
 import {
   DeleteSupplierMasterRequestPayload,
   PostSupplierMasterRequestPayload,
   PutSupplierMasterRequestPayload,
 } from 'stores/types/supplierMasterType';
+import { getRequest } from 'utils/apiClient';
 
 const clearValues: ISupplierMaster = { id: undefined, name: '' };
 
 function SupplierMasterPage() {
+  const DEFAULT_OFFSET = 10;
+  const FIRST_PAGE = 1;
+  const EMPTY_SEARCHER = '';
+  const supplierUrl = 'supplier/';
```

Comment:
This one can be exported from `stores/reducers/supplierMasterReducer.ts` - if url will change we will need to change it only in one place